templatevars.json

```diff
@@ -28,6 +28,20 @@
                     "userId": "{{ test1.userId }}"
                 }
             }
+        },
+        {
+            "name": "test3",
+            "request": {
+                "method": "POST",
+                "url": "/users",
+                "body": "{{ test1.userId }}"
```

Comment:
Is using a template variable in the _request_ body intentional here? Or is it intended to be a simple string?

Reply:
Sorry I meant to do this in the response body, but it was before I fixed the string issue. I'll change it.